purchase_work_acceptance_late_fine/models/res_company.py

```diff
@@ -0,0 +1,13 @@
+# Copyright 2020 Ecosoft Co., Ltd. (http://ecosoft.co.th)
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl).
+
+from odoo import fields, models
+
+
+class Company(models.Model):
+    _inherit = "res.company"
+
+    wa_fine_rate = fields.Monetary(string="Rate",)
```

Comment:
Rate is, so it should be float.
```suggestion
    wa_fine_rate = fields.Float(
        string="Rate",
    )
```